apps/extending-graphql-queries-server/src/order/dtos/OrderFindManyExtendedWhereArgs.ts

```diff
@@ -0,0 +1,62 @@
+/*
+------------------------------------------------------------------------------ 
+This code was generated by Amplication. 
+ 
+Changes to this file will be lost if the code is regenerated. 
+
+There are other ways to to customize your code, see this doc to learn more
+https://docs.amplication.com/how-to/custom-code
+
+------------------------------------------------------------------------------
+  */
+import { ArgsType, Field } from "@nestjs/graphql";
+import { ApiProperty } from "@nestjs/swagger";
+import { IsOptional, ValidateNested, IsInt } from "class-validator";
+import { Type } from "class-transformer";
+import { OrderWhereInputWithExtendedCustomer } from "./OrderWhereInputWithExtendedCustomer";
+import { OrderOrderByInput } from "../base/OrderOrderByInput";
+
+@ArgsType()
+class OrderFindManyExtendedWhereArgs {
```

Comment:
I think it is better to think of name that will provide more context instead of just using "extended"